build/bootstrap/bootstrap-debian.sh

```diff
@@ -27,18 +27,14 @@ sudo apt-get install -y --no-install-recommends \
   g++ \
   git \
   nodejs \
-  bison
+  bison \
+  protobuf-compiler
```

Comment:
Why this change? The `README` [says](https://github.com/cockroachlabs/managed-service#install-bazel) that you should NOT have `protoc` installed globally.

Reply:
If I am not wrong, this is for mac machine. Right above [this line](https://github.com/cockroachlabs/managed-service?tab=readme-ov-file#install-google-cloud-sdk) readme says to install globally using `brew`.

Reply:
The `README` clearly says that having `protoc` globally installed breaks the build. 

1. Why do you want to install it?
2. If you're going to go against what the `README` says, did you at least test whether what it says is accurate?

Reply:
I don't believe it needs to be installed globally.

Reply:
I have removed this, and pushed commit with all the suggested changes. In the background I will be checking if skipping `protobuf` is giving any issue with cockroach and managed-service development.